index.html

```diff
@@ -216,6 +216,57 @@ <h3 class="skill-heading">Skills</h3>
           </ul>
         </div>
       </section>
+      <footer>
+        <div class="container container-information">
+           <div class="top-footer">
+              <h2 class="footer-heading">Get Started</h2>
+              <p class="last-text">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna.</p>
+              <form>
+                <ul>
+                  <li class="input-item">
+                    <label class="label1">Your name</label>
+                    <input class="input2" type="text" id="name" name="user-name" required maxlength="30">
+                  </li>
+                  <li class="input-item">
+                    <label class="label1">Your email</label>
+                    <input class="input2" type="email" id="mail" name="user-email" required>
+                  </li>
+                  <li class="input-item">
+                    <label>message</label>
+                    <textarea id="textarea1" placeholder="Enter your message here..." required maxlength="500"></textarea>
```

Comment:
- [ ] kindly change the font color when typing data inside form elements, currently they are barely visible try something darker to make it easily visible

Before          |  After
:-------------------------:|:-------------------------:
![texb](https://user-images.githubusercontent.com/89282221/187510120-9e941122-7fb0-404b-a07d-9045b05b6acc.gif) |  ![texa](https://user-images.githubusercontent.com/89282221/187510140-cbb6878f-48cd-440c-a969-7c86a10591fc.gif)